crypto/sigpb/sigpb.proto

```diff
@@ -46,24 +46,10 @@ message DigitallySigned {
     ECDSA = 3;
   }
 
-  // SignatureCipherSuite defines the set of algorithms used for signing.
-  enum SignatureCipherSuite {
-    // Signature and hash algorithm are unknown / not set. 
-    UNKNOWN_CIPHER_SUITE = 0;
-    // RSA with SHA256.
-    RSA_SHA256 = 1;
-    // ECDSA with SHA256.
-    ECDSA_SHA256 = 2;
-  }
-
   // hash_algorithm contains the hash algorithm used.
-  // TODO(gdbelvin): deprecate in favor of cipher_suite.
   HashAlgorithm hash_algorithm = 1;
   // sig_algorithm contains the signing algorithm used.
-  // TODO(gdbelvin): deprecate in favor of cipher_suite.
   SignatureAlgorithm signature_algorithm = 2;
-  // cipher_suite contains the combination of ciphers used for this signature.
-  SignatureCipherSuite signature_cipher_suite = 4;
```

Comment:
This probably shouldn't have been here in the first place, given that this `message` is supposed to be 1-1 with `DigitallySigned` from [RFC 5246 §4.7](https://tools.ietf.org/html/rfc5246#section-4.7); however, removing a field is a [bit of a worry](https://developers.google.com/protocol-buffers/docs/proto3#updating)  -- are we absolutely sure this field has never been used anywhere?

Reply:
yes, we're absolutely sure. These fields were never set.